monitoring/prometheus-rules/vm.yaml

```
@@ -0,0 +1,21 @@
+- name: kubernetes.virtualization.vm
+  rules:
+    - alert: D8VirtualizationVMFirmwareOutOfDate
+      expr: d8_virtualization_virtualmachine_firmware_up_to_date == 0
```

Mb something like this?

```suggestion
      expr: count(d8_virtualization_virtualmachine_firmware_up_to_date != 0) > 0
```